### ISSUE TITLE
max_n seems complicated

### ISSUE THREAD

**Issue by kootenpv** [kriadmin/30-seconds-of-python-code#3]
For this code:

```python
from copy import deepcopy


def max_n(arr, n=1):
    numbers = deepcopy(arr)
    numbers.sort()
    numbers.reverse()
    return numbers[:n]

```
https://github.com/kriadmin/30-seconds-of-python-code#max_n

Why not just do:

```python
def max_n(arr, n=1, reverse=True):
    return sorted(arr, reverse=reverse)[:n]
```

In fact, this seems so basic you wouldn't need a snippet for it?

**Comment by gerardlouw** [comment]
Why reinvent the wheel? The standard library already has this (and it has a better time complexity).
`heapq.nlargest(n, arr)`
See also `heapq.nsmallest`.

**Comment by kriadmin** [comment]
@gerardlouw If you would have looked at the snippets most of them are already implemented in the standard library. This snippets not for someone to use in production but just to learn the pythonic way to do things.

**Comment by kriadmin** [comment]
Fixed via #30

**Comment by bsutherland** [comment]
> @gerardlouw If you would have looked at the snippets most of them are already implemented in the standard library. This snippets not for someone to use in production but just to learn the pythonic way to do things.

Some other snippets mention existing implementations in the standard library..
